app/src/main/java/com/worldsnas/starplayer/di/Androidmodule.kt

```diff
@@ -0,0 +1,12 @@
+package com.worldsnas.starplayer.di
+
+import android.app.Application
+import android.content.Context
+import dagger.Module
+
+@Module
+class Androidmodule (private val application: Application){
```

Comment:
lotfan az pass dadane maghadir be module ha dori konid
mitonid bejash az `@BindsInstance` estefade konid

Reply:
in file hamchenan niaze?